node/src/peer_tracker.rs

```diff
@@ -207,30 +206,20 @@ impl PeerTracker {
         }
     }
 
-    /// Sets peer as identified.
-    pub fn set_identified(&self, peer: PeerId, info: &identify::Info) {
-        let mut peer_info = self.get(peer);
-
-        for addr in &info.listen_addrs {
-            if !peer_info.addrs.contains(addr) {
-                peer_info.addrs.push(addr.to_owned());
-            }
-        }
-
-        peer_info.state = PeerState::Identified;
-    }
-
     /// Returns true if peer is connected.
+    #[allow(dead_code)]
```

Comment:
why do we need this?

Reply:
I guess `pub(crate)` could do

Reply:
`is_connected` is not really used. even with `pub(crate)` we have a warning on the compilation, but I didn't want to remove the functionality.